lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
Yes this is going to work on how Fake JSON  API is and with post and collections and post returns a single object and inside of each object is the array of data